### ISSUE TITLE
Conflict with Up_And_Down_And_All_Around

### ISSUE THREAD

**Issue by velocityraptor-real** [TeamGalacticraft/Galacticraft#15]
Minecraft version: 1.12.2
Galacticraft version ("latest" is not a valid option): 4.0.2.281
Single player (SP)
No addons, just the 3 base Galacticraft mods and Up_And_Down_And_All_Around
Downloadsite: https://micdoodle8.com/mods/galacticraft/downloads

When using a gravity anchor from Up_And_Down_And_All_Around, the new gravity rules apply and the camera rotates, but the cameraposition stays in the vanilla head position. The vanilla player collisionbox also does not get removed

![2021-01-04_16 20 32](https://user-images.githubusercontent.com/56190563/103550314-01b4d580-4ea9-11eb-8e9f-621e0e03097a.png)

https://www.youtube.com/watch?v=ueoRnqe-Nko

**Comment by necauqua** [comment]
This is caused by the developers not mixing into entity player classes for their changes but rather completely replacing them with their own (for unknown reasons, maybe because their code is old and mixin a thing back then or something), so mixins for these classes from other mods (obviously including the one you've mentioned) do not matter - so the camera stays at vanilla (or, rather, galacticraft) height.

tl;dr; one of the two mods should do something for compatibility, preferrably galacticraft, but likely they wont do that, given that they are humans

**Comment by ROMVoid95** [comment]
Since the issue stems from the action performed by Up_And_Down_And_All_Around, Closing as the incompatibility is on their end.
